lib/processLayerStyles.js

```diff
@@ -72,10 +80,61 @@ const layerFill = (layerStyle) => {
 
         }
 
+        if (fill.fillType == FILL_TYPE.IMAGE && fill.isEnabled == true) {
+          // Copy the image from the sketch file to the output folder
```

Comment:
probably out of scope, but layerFill is now quite large - would it be possible to break it down into smaller functions for readability?

Reply:
I'll do that in another PR